### PR TITLE
Improve creation of event to avoid fail to save sometimes.

### PR DESCRIPTION
**Pull request by xmano**
Fix intermittent problem with saving of event.

### CODE REVIEW

File: apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts [hunk 1]
```diff
@@ -25,13 +25,19 @@ export default class EventPage {
...
-              screeningName: string, isPublic: boolean = false) {
+              screeningName: string, isPublic: boolean = false,
+              inviteeList:string[] = []) {
     cy.log(`createEvent : {${eventTitle}}`);
     const event: EventEditPage = this.createDetailedEvent(eventDate, 'Screening', eventTitle);
     event.uncheckPrivate(isPublic);
 
     event.selectMovie(screeningName);
-
+    if (inviteeList.length != 0) {
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts:35]: use `!==`

File: apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts [hunk 2]
```diff
@@ -47,18 +47,10 @@ describe('Organiser invites other users to private screening', () => {
     const p1 = new FestivalDashboardHomePage();
     const p2: EventPage = p1.goToCalendar();
     cy.log(`Create screening {${TestEVENT.event}}`)
-    const p3: EventEditPage = p2.createDetailedEvent(NOW, 'Screening', TestEVENT.event);
-    p3.addEventTitle(TestEVENT.event);
-    p3.checkAllDay();
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts:52]: `checkAllDay` seems not to be used anymore. you can remove it

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/support/pages/dashboard/EventPage.ts:35]: ok

File: apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts (hunk 2 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts:52]: Will retain it until all tests are reworked. We might need it for some scenarios and hence keeping it for now.

**Pull request merged**